dashboard/test/ui/features/hamburger.feature

```diff
@@ -41,76 +40,79 @@ Feature: Hamburger dropdown
-    Then I wait to see "#hamburger-icon"
-    And I click selector "#hamburger-icon"
-    Then I wait to see "#hamburger-contents"
+    Then I wait to see "#help-button"
+    And I click selector "#help-button"
+    Then I wait to see "#help-contents"
     And I see "#gamelab-docs"
 
   Scenario: Student viewing hamburger dropdown in English on desktop on level
     Given I create a student named "Sally Student"
     And I am on "http://studio.code.org/s/allthethings/stage/1/puzzle/1"
     Then I wait until I am on "http://studio.code.org/s/allthethings/stage/1/puzzle/1"
     Then I wait to see "#hamburger-icon"
+    Then I wait to see "#help-button"
     And I click selector "#hamburger-icon"
     Then I wait to see "#hamburger-contents"
     And I see ".divider#after-student"
-    And I see "#report-bug"
-    And I see "#support"
-    And I see ".divider#before-pegasus"
     And I see "#about_entries"
     And I see "#educate_entries"
     And I see "#learn"
     And I see "#stats"
     And I see "#help-us"
+    Then I click selector "#help-button"
+    Then I wait to see "#help-contents"
+    And I see "#report-bug"
+    And I see "#support"
+  
 
   Scenario: Teacher viewing hamburger dropdown in English on desktop on level
     Given I create a teacher named "Tessa Teacher"
     And I am on "http://studio.code.org/s/allthethings/stage/1/puzzle/1"
     Then I wait until I am on "http://studio.code.org/s/allthethings/stage/1/puzzle/1"
     Then I wait to see "#hamburger-icon"
+    Then I wait to see "#help-button"
     And I click selector "#hamburger-icon"
     Then I wait to see "#hamburger-contents"
     And I see ".divider#after-teacher"
-    And I see "#report-bug"
-    And I see "#support"
-    And I see "#teacher-community"
-    And I see ".divider#before-pegasus"
     And I see "#about_entries"
     And I see "#educate_entries"
     And I see "#learn"
     And I see "#stats"
     And I see "#help-us"
-
+    Then I click selector "#help-button"
+    Then I wait to see "#help-contents"
+    And I see "#report-bug"
+    And I see "#support"
+    And I see "#teacher-community"
+ 
 Scenario: Signed out user viewing hamburger dropdown in Spanish on desktop
   Given I am on "http://code.org/lang/es"
   Then I wait until I am on "http://code.org/"
   And I dismiss the language selector
-  Then I wait to see "#hamburger-icon"
-  And I click selector "#hamburger-icon"
-  Then I wait to see "#hamburger-contents"
+  Then I wait to see "#help-button"
+  Then I click selector "#help-button"
+  Then I wait to see "#help-contents"
   And I see "#report-bug"
   And I see "#support"
   Then element "#teacher-community" is not visible
-  Then element "#learn" is not visible
-  Then element ".divider#before-pegasus" is not visible
-  Then element "#learn" is not visible
-  Then element "#educate_entries" is not visible
-  Then element "#about_entries" is not visible
-  Then element "#stats" is not visible
-  Then element "#help-us" is not visible
```

Comment:
why remove these?

Reply:
The hamburger menu disappears when viewing in non-English on desktop because the only things in the hamburger menu are the help items.